assets/models/system/openai-clip-image-text-embeddings-vit-base-patch32/model.yaml

```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: finetuning-image-models
+  container_path: HuggingFace-mlflow/image-text-embeddings-openai-clip-vit-base-patch32/1696630981/mlflow_model_folder
```

Comment:
Should we update this path also to align with the name change ?

Reply:
[publishing pipeline](https://ml.azure.com/experiments/id/0534f38d-31dc-41cf-9ee4-badae3b6fce1/runs/stoic_wire_l6vjc9tfck?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/rbhimani-rg/providers/Microsoft.MachineLearningServices/workspaces/rbhimani-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#) used to get the new container path